vim_menutrans_helper.py

```diff
@@ -0,0 +1,228 @@
+#!/user/bin/env python3
+
+import re
+import glob
+import os
+import sys
+
+menu_commands = {"me", "menu", "noreme", "noremenu", "am",
+             "amenu", "an", "anoremenu", "nme", "nmenu",
+             "nnoreme", "nnoremenu", "ome", "omenu", "onoreme",
+             "onoremenu", "vme", "vmenu", "vnoreme", "vnoremenu",
+             "xme", "xmenu", "xnoreme", "xnoremenu", "sme", "smenu",
+             "snoreme", "snoremenu", "ime", "imenu", "inoreme", "inoremenu",
+             "cme", "cmenu", "cnoreme", "cnoremenu", "tlm", "tlmenu", "tln",
+             "tlnoremenu", "tunmenu", "tu", "popup", "popu"}
```

Comment:
I think it helps to include some references to Vim's document to clarify what these strings are from. How about this:

```python
MENU_COMMANDS = {
    # Menu-related commands. See ":help creating-menus"

    # Regular menu.
    "me", "menu", "noreme", "noremenu",
    # All modes except Terminal.
    "am", "amenu", "an", "anoremenu",
    # Normal mode.
    "nme", "nmenu", "nnoreme", "nnoremenu", "nunme", "nunmenu",
    # Operator-pending mode.
    "ome", "omenu", "onoreme", "onoremenu", "ounme", "ounmenu",
    # Visual mode. 
    "vme", "vmenu", "vnoreme", "vnoremenu", "vunme", "vunmenu",
    # Visual and Select mode.
    "xme", "xmenu", "xnoreme", "xnoremenu", "xunme", "xunmenu",
    # Select mode.
    "sme", "smenu", "snoreme", "snoremenu", "sunme", "sunmenu",
    # Insert mode.
    "ime", "imenu", "inoreme", "inoremenu", "iunme", "iunmenu",
    # Cmdline mode.
    "cme", "cmenu", "cnoreme", "cnoremenu", "cunme", "cunmenu",
    # Terminal mode.
    "tlm", "tlmenu", "tln", "tlnoremenu", "tlu", "tlunmenu",

    # Popup menu definition. See ":help :popup".
    "popup", "popu"
}
```